python/trainers/ppo_trainer.py

```diff
@@ -46,15 +46,16 @@ def __init__(self, sess, env, brain_name, trainer_parameters, training):
             self.m_size = env.brains[brain_name].memory_space_size
             self.sequence_length = trainer_parameters["sequence_length"]
         if self.use_recurrent:
-            if (self.m_size == 0):
+            if self.m_size == 0:
                 raise UnityTrainerException("The memory size for brain {0} is 0 even though the trainer uses recurrent."
                                             .format(brain_name))
-            elif (self.m_size % 4 != 0):
+            elif self.m_size % 4 != 0:
                 raise UnityTrainerException("The memory size for brain {0} is {1} but it must be divisible by 4."
                                             .format(brain_name, self.m_size))
 
         self.variable_scope = trainer_parameters['graph_scope']
         with tf.variable_scope(self.variable_scope):
+            tf.set_random_seed(seed)
```

Comment:
So in the master `learn.py` you check if `seed != -1` before setting the seed for `tf`. However PPOTrainer is again called with the `seed` which may potentially be `-1` which means that here we could end up setting `tf.set_random_seed(-1)` ?